src/siembol/templates/configmap-manager.yaml

```diff
@@ -0,0 +1,9 @@
+apiVersion: v1
```

Comment:
not needed if we use config from docker image

Reply:
@of13tch @jgiannuzzi I'd love your thoughts here.  I thought we needed these `ConfigMap` -- they don't feel like things that you can bake into the docker image itself.

Reply:
Addressing in a later issue